backend/models/users.js

```diff
@@ -32,7 +32,7 @@ const users = {
 
     const fetchString = "SELECT id, email FROM users WHERE id = ?;";
```

Comment:
role_id was missing here, so it was missing from the tokenPayload as well
```suggestion
    const fetchString = `
      SELECT id, email, role_id 
      FROM users
      WHERE id = ?;
      `;
```